src/witan/models/dem/ccm/fert/hist_asfr_ws.clj

```diff
@@ -7,8 +7,8 @@
 
 ;; GET DATA FOR ROOT NODES
 (def root-data-paths {:births-data "resources/test_data/bristol_births_data.csv"
-                 :at-risk-popn "resources/test_data/bristol_denominators.csv"
-                 :mye-coc "resources/test_data/bristol_mye_coc.csv"})
+                      :at-risk-popn "resources/test_data/bristol_denominators.csv"
```

Comment:
the population at risk will be changed to be the starting population as agreed with Ben, but we will address that when we come to code the historic ASFR for real